pkg/notify/opsgenie/opsgenie.go

```diff
@@ -0,0 +1,67 @@
+package opsgenie
+
+import (
+	"github.com/juju/errors"
+	"github.com/opsgenie/opsgenie-go-sdk-v2/alert"
+	"github.com/opsgenie/opsgenie-go-sdk-v2/client"
+
+	"github.com/ovh/utask/pkg/notify"
+)
+
+const (
+	// Type represents OpsGenie as notify backend
+	Type = "opsgenie"
+
+	// Zone are opsgenie api zones
+	ZoneSandbox = "sandbox"
+	ZoneDefault = "global"
+	ZoneEU      = "eu"
+)
+
+// NotificationSender is a notify.NotificationSender implementation
+// capable of sending formatted notifications over OpsGenie (https://www.atlassian.com/software/opsgenie)
+type NotificationSender struct {
+	opsGenieZone   string
+	opsGenieAPIKey string
+	client         *alert.Client
+}
+
+// NewOpsGenieNotificationSender instantiates a NotificationSender
+func NewOpsGenieNotificationSender(zone, apikey string) (*NotificationSender, error) {
+	zonesToApiUrls := map[string]client.ApiUrl{
+		ZoneDefault: client.API_URL,
+		ZoneEU:      client.API_URL_EU,
+		ZoneSandbox: client.API_URL_SANDBOX,
+	}
+	apiUrl, present := zonesToApiUrls[zone]
+	if !present {
+		return nil, errors.NotFoundf("opsgenie zone %q not found", zone)
```

Comment:
```suggestion
		return nil, errors.NotFoundf("opsgenie zone %q", zone)
```

juju/errors already suffix with `not found`